libdebug/data/symbol.py

```diff
@@ -0,0 +1,33 @@
+#
+# This file is part of libdebug Python library (https://github.com/libdebug/libdebug).
+# Copyright (c) 2024 Gabriele Digregorio. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for details.
+#
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+
+@dataclass
+class Symbol:
+    """A symbol in the target process.
+
+    start (int): The start address of the symbol in the target process.
+    end (int): The end address of the symbol in the target process.
+    name (str): The name of the symbol in the target process.
+    backing_file (str): The backing file of the symbol in the target process.
+    """
+
+    start: int
```

Comment:
Can we get `base` as an alias for `start`?

Reply:
done :)